pkg/skaffold/build/local/local_test.go

```diff
@@ -211,8 +331,23 @@ func TestLocalRun(t *testing.T) {
 			api: testutil.FakeAPIClient{
 				ErrImageInspect: true,
 			},
-			tags:      tag.ImageTags(map[string]string{"gcr.io/test/image": "gcr.io/test/image:tag"}),
-			shouldErr: true,
+			tags: tag.ImageTags(map[string]string{"gcr.io/test/image": "gcr.io/test/image:tag"}),
+			expectedResults: []testResult{
+				{
+					buildResult: build.Result{
+						Target: &latest.Artifact{
+							ImageName: "gcr.io/test/image",
+							ArtifactType: latest.ArtifactType{
+								DockerArtifact: &latest.DockerArtifact{
+									CacheFrom: []string{"pull"},
+								},
+							},
+						},
+						Error: fmt.Errorf("building [gcr.io/test/image]: build artifact: pulling cache-from images: getting imageID for pull: inspecting image: "),
```

Comment:
this is going to be fragile - can we match on the important part of the error that we are interested in?

Reply:
changed it to match the provided error prefix (which contains artifact info and shouldn't change)